src/LoggerModem.cpp

```diff
@@ -63,6 +64,11 @@ String loggerModem::getModemName(void) {
     return _modemName;
 }
 
+String loggerModem::getModemDevId(void) {
+    return _modemName + F(" Sn ") + _modemSerialNumber + F(" HwVer ") +
+        _modemHwVersion + F(" FwVer ") + _modemFwVersion;
+}
+
```

Comment:
These need to be populated for all modems or moved just to the XBee where they are populated.